pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -3,8 +3,10 @@ title: Included in the accessibility tree
 key: included-in-the-accessibility-tree
 ---
 
-Elements included in the accessibility tree of platform specific accessibility APIs. Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+DOM nodes included as nodes in the accessibility tree of platform specific accessibility APIs. 
 
-The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are [included in the accessibility tree][] can be found in the [HTML accessibility API mappings](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings](https://www.w3.org/TR/svg-aam/).
+Nodes in the accessibility tree are exposed to assistive technologies, allowing users to interact with the object in a way that meets the requirements of the individual user.
```

Comment:
You are talking about "Nodes" = plural. Maybe keep the plural for the rest?

> interact with the object

plural:

> interact with objects